examples/colorsetter/colorsetter.ino

```diff
@@ -0,0 +1,74 @@
+/*
+* The example demonstrates the operation of all RGBEncoder inner devices.
+* The encoder adjusts the RGB color components one by one. 
+* The button selects which color component is adjusted.
+* The LED displays the selected color.
+*/
+
+// Include library
+#include "RGBEncoder.h"
+
+// Create an RGBEncoder object
+RGBEncoder knob;
+
+// mode is the state of a finite-state machine.
+// By switching these states, we select the color component that we want to adjust.
+#define CHANGE_RED 0
+#define CHANGE_GREEN 1
+#define CHANGE_BLUE 2
+byte mode = CHANGE_RED;
+
+// Current color components values
+byte red = 0, green = 0, blue = 0;
+
+// Handler of button
+void buttonHandler(bool state) {
+    if (state) { // if button pressed ...
```

Comment:
state -> pressed.

Prefer early exit to remove nesting:

```cpp
void buttonHandler(bool pressed) {
    if (!pressed)
        return;

    switch (mode) {
    // ...
```

Reply:
Done